garage/contrib/exp/experiment.py

```diff
@@ -0,0 +1,58 @@
+import gym
+
+from garage.contrib.exp import Agent
+from garage.contrib.exp import Checkpointer
+from garage.contrib.exp import Logger
+from garage.contrib.exp import Sampler
+
+
+class Experiment():
+    def __init__(
+            self,
+            env: gym.Env,
```

Comment:
let's hold off on type annotations until we actually decide whether to use them in the codebase.